docker-compose.yaml

```diff
@@ -0,0 +1,24 @@
+slackbot:
+  image: node:6
+  entrypoint: /app/entrypoint.sh
+  environment:
+    - DATABASE_URL=postgres://postgres:standup@database/standup
+    - PORT=3000
+    - LOG_LEVEL=10
+  expose:
+    - 3000
```

Comment:
Actually, let's leave it in.  That way when the API is added, it'll already be exposed if we want to link it up to a consuming container.  :smile: